samples/MauiEmbedding/Directory.Build.props

```diff
@@ -18,6 +18,8 @@
 		<IsMacCatalyst>false</IsMacCatalyst>
 		<IsWinAppSdk>false</IsWinAppSdk>
 		<_UseTelerik>false</_UseTelerik>
+        <!-- Remove after https://github.com/unoplatform/uno/pull/13034 is merged -->
```

Comment:
please make sure the whitespace characters are the same and not mixing spaces / tabs

Reply:
this isn't required